vite.config.js

```diff
@@ -52,14 +53,29 @@ export default defineConfig({
         include: ["test/**/*.test.{js,mjs,cjs}"],
         environment: "jsdom",
         setupFiles: ["test/setup.js"],
-        root: '.',
+        root: ".",
     },
-    plugins: [vue(), serveLocalesPlugin()],
+    plugins: [
+        vue(),
+        serveLocalesPlugin(),
+        copy({
+            targets: [
+                { src: ["locales", "resources", "src/tabs", "src/images"], dest: "src/dist" },
```

Comment:
Hmm, guessing we can't have `/dist` in the `root` because of `root` setting in vite config 🤔 Having it in `src` feels like will backfire

Reply:
interesting and good catch. but it is ignored by default (`.gitignore `)

Reply:
It was already set up that way in #3292. But yeah in `src` it will probably come back to bite us. What do you suggest?

Reply:
Should we look into

![image](https://github.com/betaflight/betaflight-configurator/assets/8344830/d165d70a-b1ac-47eb-a562-34b6ec6a3f99)

Reply:
> Should we look into

i don't really understand the errors. they do go away upon re-execution. they tend to only present themselves with new code/execution